conda/models/records.py

```diff
@@ -636,9 +648,6 @@ class PrefixRecord(PackageRecord):
 
     # app = ComposableField(App, required=False)
 
-    #: str: The :class:`MatchSpec` that the user requested or ``None`` if dependency it was installed as a dependency.
-    requested_spec = StringField(required=False)
```

Comment:
Does this need to be deprecated from `PrefixRecord` before removal?

Reply:
`PrefixRecord` inherits from `SolvedRecord`, so we didn't really remove it.